llvm/lib/Target/AArch64/MCTargetDesc/AArch64ELFStreamer.cpp

```diff
@@ -271,8 +271,7 @@ class AArch64ELFStreamer : public MCELFStreamer {
   }
 
   void emitMappingSymbol(StringRef Name) {
-    auto *Symbol = cast<MCSymbolELF>(getContext().getOrCreateSymbol(
-        Name + "." + Twine(MappingSymbolCounter++)));
+    auto *Symbol = cast<MCSymbolELF>(getContext().createLocalSymbol(Name));
```

Comment:
I think the removed code is the only use for MappingSymbolCounter, can it be removed?

Reply:
Removed by the second commit. Thanks